can_tools/scrapers/base.py

```diff
@@ -402,6 +440,26 @@ def validate(self, df, df_hist):
         validated : bool
             Whether we have validated the data
         """
+        if is_time_series(df):
+            self._validate_time_series(df)
+
+        cats = [
+            # ("total_vaccine_initiated", "total_vaccine_doses_administered"),
+            ("total_vaccine_completed", "total_vaccine_doses_administered"),
+            ("total_vaccine_completed", "total_vaccine_initiated"),
+        ]
+        cumulatives = df.query("measurement == 'cumulative'")
+        categories = list(cumulatives["category"].unique())
+        for cat_small, cat_large in cats:
+            if cat_small in categories and cat_large in categories:
+                ok = cat1_ge_cat2(
+                    cumulatives,
+                    cat_large,
+                    cat_small,
+                    drop_levels=["unit", "category"],
+                )
+                if not ok:
+                    raise ValidateDataFailedError(f"{cat_small} >= {cat_large}")
         return True
```

Comment:
With validate functions I think we either want to always return a bool (i.e. true if no exception raised, or false if one is raised) or make it more of a `check` method and not return anything but raise an exception if there's a failed check.  Alternatively, I've seen examples of returning a list of errors encountered.  

I think one advantage of exceptions here is they'll be captured by sentry - but don't feel super strongly here

Reply:
I value consistency. How would we return "false if one [exception] is raised"? Once we get an exception we give up control flow and can't provide a return value.

Reply:
I'm thinking maybe we should just not return anything?